release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -119,6 +119,46 @@ With this release, migration from the OpenShift SDN network plugin to OVN-Kubern
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
```

Comment:
According to google, there is an "Azure Files" product with an s, but maybe that's something different - just checking.

Reply:
I believe that that is referring to something else.